generate.sh

```diff
@@ -224,19 +285,45 @@ function CreateProject {
     <IsAotCompatible>true</IsAotCompatible>
   </PropertyGroup>
   <ItemGroup>
+    <!-- iOS is arm64 only -->
+    <Content Include="./SwiftBindings_arm64.xcframework/ios-arm64/**">
+        <PackagePath>runtimes/ios-arm64/native/</PackagePath>
+        <Pack>true</Pack>
+    </Content>
+    <Content Include="./SwiftBindings_arm64.xcframework/ios-arm64-simulator/**">
+        <PackagePath>runtimes/iossimulator-arm64/native/</PackagePath>
+        <Pack>true</Pack>
+    </Content>
+    <Content Include="./SwiftBindings_x86_64.xcframework/ios-x86_64-simulator/**">
+        <PackagePath>runtimes/iossimulator-x64/native/</PackagePath>
+        <Pack>true</Pack>
+    </Content>
+    <!-- tvOS is arm64 only -->
+    <Content Include="./SwiftBindings_arm64.xcframework/tvos-arm64/**">
+        <PackagePath>runtimes/tvos-arm64/native/</PackagePath>
+        <Pack>true</Pack>
+    </Content>
+    <Content Include="./SwiftBindings_arm64.xcframework/tvos-arm64-simulator/**">
+        <PackagePath>runtimes/tvossimulator-arm64/native/</PackagePath>
+        <Pack>true</Pack>
+    </Content>
+    <Content Include="./SwiftBindings_x86_64.xcframework/tvos-x86_64-simulator/**">
+        <PackagePath>runtimes/tvossimulator-x64/native/</PackagePath>
+        <Pack>true</Pack>
+    </Content>
     <Content Include="./SwiftBindings_arm64.xcframework/macos-arm64/**">
         <PackagePath>runtimes/osx-arm64/native/</PackagePath>
         <Pack>true</Pack>
     </Content>
-    <Content Include="./SwiftBindings_x64.xcframework/macos-64/**">
+    <Content Include="./SwiftBindings_x86_64.xcframework/macos-x86_64/**">
```

Comment:
This won't handle symlinks correctly, because nupkgs don't support symlinks (nupkgs with symlinks aren't unzipped correctly, the symlinks aren't preserved).

Zipping up `./SwiftBindings_x86_64.xcframework` into `SwiftBindings_x86_64.xcframework.zip` and adding that zip file to the nupkg is the fix.

Reply:
Updated, thanks. Just to confirm:
 - The .zip file containing multiple targets is included in the NuGet package
 - During app build, the .zip file is copied into the app bundle
- The iOS SDK unzips the xcframework and places the target-specific framework into the bundle

Reply:
Step 2 (copying the zip file) doesn't happen because it's not necessary, we unzip directly from the nuget on disk, but otherwise yes, that's what's supposed to happen.

It should be fairly easy to verify it works as well: just reference the nupkg from an iOS or MacCatalyst project, make sure to add a reference in code to the nuget and build.